chia/types/eligible_coin_spends.py

```diff
@@ -229,6 +228,11 @@ def get_deduplication_info(
         self.deduplication_spends.update(new_dedup_spends)
         return unique_coin_spends, uint64(cost_saving), unique_additions
 
+
+@dataclasses.dataclass(frozen=True)
```

Comment:
I don't understand how this class can be frozen (immutable). Surely you add elements to the `fast_forward_spends` dictionary, which (presumably) requires the class to be mutable.
(I also don't understand how `IdenticalSpendDedup` can be frozen)

Reply:
While the dataclass being frozen prevents reassigning this map, the map itself is and remains mutable.